marshaling.go

```diff
@@ -360,56 +363,56 @@ func (ctx *structScanCtx) convert(fieldCtx structScanFieldCtx, value Value) (ref
 	return value.structScanConvert(ctx, fieldCtx)
 }
 
-func (value *Undefined) structScanConvert(_ *structScanCtx, ctx structScanFieldCtx) (reflect.Value, error) {
-	if ctx.destType.Kind() != reflect.Ptr {
-		ctx.sourceType = value.Type()
-		return ctx.cannotConvert(fmt.Sprintf("dest must be a *%s", ctx.destType.Name()))
+func (value *Undefined) structScanConvert(ctx *structScanCtx, fieldCtx structScanFieldCtx) (reflect.Value, error) {
+	if !ctx.allowUndefinedToNonPointer && fieldCtx.destType.Kind() != reflect.Ptr {
```

Comment:
cool! i see types that use custom converters or overrides wouldn't hit this path, but i suppose that's the whole point of having custom converts ;-) https://github.com/helloeave/worksheets/blob/master/marshaling.go#L323